src/aap_eda/api/serializers/activation.py

```diff
@@ -357,6 +434,7 @@ class PostActivationSerializer(serializers.ModelSerializer):
         validators=[validators.check_if_extra_var_exists],
     )
     awx_token_id = serializers.IntegerField(
+        required=False,
```

Comment:
@Alex-Izquierdo @jshimkus-rh This is a nasty one it happens because we are passing in the EventStream object as Activation object. Once we resolve https://issues.redhat.com/browse/AAP-20044 and call the appropriate validate function in EventStream or[ activation here](https://github.com/ansible/eda-server/blob/eb2df468cb2288fda08772cf83ec4464bb4b10d0/src/aap_eda/services/activation/manager.py#L202) it should resolve itself.

Reply:
A TODO comment would be good